### PR TITLE
Nikita/nh model

### PR DESCRIPTION
**Pull request by nikitr**
Completed #6 and #7. Unsure if UserNeighborhood group with has_many :through relationship was done correctly.

### CODE REVIEW

File: log/development.log [hunk 1]
```diff
@@ -0,0 +1,605 @@
...
+  [1m[36mActiveRecord::SchemaMigration Load (0.7ms)[0m  [1mSELECT "schema_migrations".* FROM "schema_migrations"[0m
+Processing by Rails::WelcomeController#index as HTML
+  Rendered /Users/nikitarau/.rvm/gems/ruby-2.2.1/gems/railties-4.2.1/lib/rails/templates/rails/welcome/index.html.erb (1.2ms)
+Completed 200 OK in 14ms (Views: 6.1ms | ActiveRecord: 0.0ms)
+
+
+Started GET "/" for ::1 at 2015-10-17 13:47:05 -0500
+Processing by Rails::WelcomeController#index as HTML
+  Rendered /Users/nikitarau/.rvm/gems/ruby-2.2.1/gems/railties-4.2.1/lib/rails/templates/rails/welcome/index.html.erb (0.0ms)
+Completed 200 OK in 2ms (Views: 1.6ms | ActiveRecord: 0.0ms)
```
Comment (hidesminimally) [inline log/development.log]: @nikitr remove all log files

File: config/secrets.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
```
Comment (hidesminimally) [inline config/secrets.yml]: NOOoOOOOoO0o0OOOOOO

check out the gdoc i posted in the av chat room, and follow them steps

Comment (nikitr) [inline config/secrets.yml]: What happened?!

Comment (hidesminimally) [inline config/secrets.yml]: delete this file

**Comment by hidesminimally** [comment]
@nikitr Forgot to mention during meeting, but try to change your PR title to something that's summarizing your changes. E.g., Create userNeighborhood and Neighborhood model

**Pull request merged**